### PR TITLE
Make error messages more helpful

### PR DESCRIPTION
**Pull request by jacobwindsor**
dcp-508

- Makes error messages more helpful (@ESapenaVentura thanks)
- Sets a standard for return value of tests
- Adds UUID to the nodes
- Updates core with UUID, and new message

### CODE REVIEW

**Comment by ESapenaVentura** [comment]
I'll push soon, but aside from the duplications, I have changed the 10x_has_more_than_2_files because it is partially overseeded by ensure_lane_index

File: graph_test_set/sequence_file_linkto_cell_suspesion.adoc [hunk 1]
```diff
@@ -14,16 +14,5 @@ This next query provides the filenames, biomaterial IDs and nucleic acid sources
 MATCH path = (b:biomaterial)-[:INPUT_TO_PROCESSES]->(p:process)<-[:DERIVED_BY_PROCESSES]-(f:sequence_file), seq_path = (p)-[:PROTOCOLS]-(l:library_preparation_protocol)
 WHERE NOT b:cell_suspension
 AND l.nucleic_acid_source STARTS WITH "single"
-RETURN f.`file_core.file_name`, b.`biomaterial_core.biomaterial_id`, l.nucleic_acid_source
+RETURN f, "File should be linked to a cell suspension", labels(p)
```
Comment (jacobwindsor) [inline graph_test_set/sequence_file_linkto_cell_suspesion.adoc:17]: Why is it labels(p) and not labels(f) @ESapenaVentura

File: graph_test_set/10x_has_more_than_2_files.adoc [hunk 2]
```diff
@@ -2,32 +2,16 @@
...
 ----
-MATCH (r:library_preparation_protocol)<-[:PROTOCOLS]-(p:process)<-[d]-(f:file)
+MATCH (r:library_preparation_protocol)<-[:PROTOCOLS]-(p:process)<-[d]-(f:sequence_file)
 WITH r, p, COUNT(d) as num_files
-WHERE r.`library_construction_method.ontology_label` =~ "10[xX] ([35]' ){0,1}v[1-3] sequencing" // 10x v2 and v3
-AND NOT 2 <= num_files <= 4 // Number of files is more than 1 and less than 5
-RETURN num_files
+WHERE r.`library_construction_method.ontology_label` =~ "10[xX] ([35]' ){0,1}v[1-3]" // 10x v1, v2 and v3
+AND NOT num_files >= 2 // Number of files is more than 1
+RETURN p, "Sequencing processes using 10x library preparation protocols must have more than 1 linked files", labels(r)
```
Comment (jacobwindsor) [inline graph_test_set/10x_has_more_than_2_files.adoc:16]: why not labels(p) @ESapenaVentura ?

File: .readme/TESTS_README.template [hunk 3]
```diff
@@ -11,11 +11,11 @@ To request the creation of new tests, please follow this guide:
...
 ---
-MATCH (r:library_preparation_protocol)<-[:PROTOCOLS]-(p:process)<-[d]-(f:file)
+MATCH (r:library_preparation_protocol)<-[:PROTOCOLS]-(p:process)<-[d]-(f:sequence_file)
 WITH r, p, COUNT(d) as num_files
-WHERE r.`library_construction_method.ontology_label` =~ "10[xX] ([35]' ){0,1}v[1-3] sequencing" // 10x v2 and v3
-AND NOT 2 <= num_files <= 4 // Number of files is more than 1 and less than 5
-RETURN num_files
+WHERE r.`library_construction_method.ontology_label` =~ "10[xX] ([35]' ){0,1}v[1-3]" // 10x v1, v2 and v3
+AND NOT num_files >= 2 // Number of files is more than 1
+RETURN p, "Sequencing processes using 10x library preparation protocols must have more than 1 linked files", labels(r)
```
Comment (jacobwindsor) [inline .readme/TESTS_README.template:18]: Why not labels(p) @ESapenaVentura ?

Comment (ESapenaVentura) [reply .readme/TESTS_README.template:18]: because I'm stoopid, fixing it!

**Review by ESapenaVentura — APPROVED** [review]
LGTM this is awesome

**Review by aaclan-ebi — APPROVED** [review]

**Review by ke4 — COMMENTED** [review]
Looks good to me. Do we need unit tests for this?

**Pull request merged**